src/scorecard.js

```diff
@@ -0,0 +1,42 @@
+function Scorecard() {
+  this._frameScore = [];
+  this._frame = 1;
+  this._gameScore = [[]]
+}
+
+Scorecard.prototype.showScore = function() {
+  return this.sumFrameScore();
+}
+
+Scorecard.prototype.enterScore = function(pins) {
+  if (this.sumFrameScore() + pins > 10) throw "maximum input per frame is 10";
```

Comment:
I like that you added an error if over 10, good thinking!

Reply:
Thanks!